frontend/src/Schedulr.js

```diff
@@ -175,6 +176,20 @@ class Schedulr extends React.Component {
     }));
   }
 
+  componentDidMount() {   
+    window.gapi.load('auth2', () => {
+      this.auth2 = gapi.auth2.init({
+        client_id: '119851197452-crnk45b5i6gsi5povitstfpd203n7j6b.apps.googleusercontent.com',
```

Comment:
Is it okay for us to include this or do we need to obfuscate it?

Reply:
I was thinking that just the client secret needed to be hidden, but I looked it up and I didn't find clearly whether the client id needed to also be hidden (it was mostly just API keys in general). I think to be safe I should probably obfuscate it, but I'll check and see if there's a definite answer. Thanks!